src/test/java/com/microsoft/azure/functions/worker/binding/tests/RpcHttpRequestDataSourceTest.java

```diff
@@ -41,14 +56,150 @@ public static RpcHttp getTestRpcHttp(Object inputBody) throws Exception {
     return httpBuilder.build();
   }
 
+  @Test
+  public void rpcHttpDataSource_To_HttpRequestMessage_NullableQueryParamsEmpty_EnvSettingEnabled() throws Exception {
```

Comment:
let's use camel-case syntax for method name, the old tests were wrote by c# expert, seems they are using c# name convention.